src/workers/userUpdatedCio.ts

```diff
@@ -0,0 +1,54 @@
+import { getFirstName, ghostUser } from '../common';
+import { TypedWorker } from './worker';
+import { cio, dateToCioTimestamp } from '../cio';
+import { User } from '../entity';
+
+function camelCaseToSnakeCase(
+  obj: Record<string, unknown>,
+): Record<string, unknown> {
+  const snakeObj: Record<string, unknown> = {};
+  for (const key in obj) {
+    snakeObj[key.replace(/[A-Z]/g, (letter) => `_${letter.toLowerCase()}`)] =
+      obj[key];
+  }
+  return snakeObj;
+}
+
+const OMIT_FIELDS: (keyof User)[] = [
+  'id',
+  'bio',
+  'devcardEligible',
+  'readme',
+  'readmeHtml',
+  'infoConfirmed',
+  'profileConfirmed',
+  'notificationEmail',
+];
```

Comment:
Does it make sense to do it other way around, specify fields we want to send? I am thinking we add some field to `User` eg. `flags` and then forget to update it we accidentally send it to customerio.

Reply:
I rather have them sent mistakenly than not send. So imagine we add experience level (like we did) and totally forget about it. Then we have to import historical data and fix the code, more like risk management

Reply:
🤝